ntbs-service/appsettings.Development.json

```diff
@@ -35,7 +35,8 @@
     "auditContext": "data source=localhost;initial catalog=ntbsAudit;trusted_connection=true;MultipleActiveResultSets=true",
     "migration": "data source=localhost;initial catalog=ntbs_migration;trusted_connection=true;MultipleActiveResultSets=true",
     "specimenMatching": "data source=localhost;initial catalog=specimen-matching;trusted_connection=true;MultipleActiveResultSets=true",
-    "reporting": "data source=localhost;initial catalog=reporting;trusted_connection=true;MultipleActiveResultSets=true"
+    "reporting": "data source=localhost;initial catalog=reporting;trusted_connection=true;MultipleActiveResultSets=true",
+    "applicationInsights": ""
```

Comment:
I'd be much happier if this was in the `ApplicationInsightsOptions` section of the config... unless there's a reason it needs to be here? (It's not really a connection string, much more of an API key)

Reply:
That was my initial thought, but it needs to go into the per-environment secret `load-test-connection-string`, and all of the other per-environment secrets live in this `ConnectionStrings` config item. I'm happy to switch it over if you think that's neater though?

Reply:
Yeah, I think that would be better

Reply:
Done!